Source/WebCore/css/calc/CSSCalcTree+Simplification.cpp

```diff
@@ -1325,6 +1325,13 @@ std::optional<Child> simplify(Anchor& anchor, const SimplificationOptions& optio
     return CanonicalDimension { .value = *result, .dimension = CanonicalDimension::Dimension::Length };
 }
 
+std::optional<Child> simplify(AnchorSize& anchorSize, const SimplificationOptions& options)
+{
+    UNUSED_PARAM(anchorSize);
```

Comment:
What's `simplify` supposed to do? and what's the difference between `simplify` and `evaluate`? (in the context of `anchor-size()`, which has nothing to simplify)

Reply:
Chatted offline with Antti to figure out what `simplify` should do.